src/main/resources/templates/java/gradle_gradle/exercise/build.gradle

```diff
@@ -10,16 +11,14 @@ dependencies {
     implementation 'org.apache.commons:commons-lang3:3.12.0'
 }
 
-def assignmentSrcDir = "src"
-
 tasks.withType(JavaCompile) {
     options.encoding = 'UTF-8'
 }
```

Comment:
Can we remove this now, since it's already defined in line 4? Or is this still needed?

Reply:
I agree. This does the same as the statement in line 4.
Unless Gradle is being weird this can be removed.